src/Sulu/Bundle/AdminBundle/Admin/Navigation/NavigationItem.php

```diff
@@ -52,14 +55,14 @@ class NavigationItem implements \Iterator
     /**
      * Contains the children of this item, which are other NavigationItems.
      *
-     * @var array
+     * @var array<self>
      */
     protected $children = [];
 
     /**
      * The current position of the iterator.
      *
-     * @var int
+     * @var int|null
```

Comment:
as discussed above let set a default value 0 so we can make sure position is always a int:

```suggestion
     * @var int```